data/json/mapgen/gas_stations/s_gas.json

```diff
@@ -287,9 +287,9 @@
         { "item": "road", "x": [ 20, 20 ], "y": [ 13, 14 ], "chance": 80, "repeat": [ 1, 4 ] },
         { "item": "camping", "x": [ 9, 9 ], "y": [ 15, 15 ], "chance": 90, "repeat": [ 1, 10 ] },
         { "item": "magazines", "x": [ 9, 9 ], "y": [ 13, 15 ], "chance": 90, "repeat": [ 1, 25 ] },
-        { "item": "softdrugs", "x": [ 8, 8 ], "y": [ 13, 15 ], "chance": 80, "repeat": [ 1, 8 ] }
+        { "item": "softdrugs", "x": [ 8, 8 ], "y": [ 13, 15 ], "chance": 80, "repeat": [ 1, 8 ] },
+        { "item": "nested_shotgun_s", "x": 3, "y": 20, "chance": 10 }
```

Comment:
Not *every* rural gas station is going to have a shotgun behind the counter.  Most of these are run by teenagers FFS.